src/lib/upload.ts

```diff
@@ -0,0 +1,38 @@
+import * as AWS from 'aws-sdk';
+import { createObjectCsvStringifier } from 'csv-writer';
+import { QueryResult } from 'pg';
+
+const s3 = new AWS.S3();
+
+export function uploadAsCsvToS3(
+	result: QueryResult,
+	bucket: string,
+	key: string,
+): Promise<number> {
+	if (result.rowCount == 0) {
+		return Promise.resolve(0);
+	}
+
+	const header = Object.keys(result.rows[0]).map((key) => ({
+		id: key,
+		title: key,
+	}));
```

Comment:
I think it's slightly cleaner to grab the keys from an example row rather than manually typing out the long list of headers